docs/dev-installation.rst

```diff
@@ -0,0 +1,74 @@
+
+Developer Installation 
+====================== 
+ 
+You can either install Girder natively on you machine or inside a virtual
+machine with Vagrant. 
+ 
+Virtual Machine 
++++++++++++++++ 
+ 
+The easiest way to develop for girder is within a virtual machine using Vagrant.
+For this, you need `Vagrant <https://www.vagrantup.com/downloads.html>`_ and `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_. 
+Girder is tested to work seamlessly with Vagrant 2.0 and VirtualBox 5.1. 
+ 
+Once you have those installed, obtain the Girder source code by cloning the Git 
+repository on `GitHub <https://github.com>`_: :: 
+ 
+    git clone https://github.com/girder/girder.git 
+    cd girder 
+ 
+Inside of the Girder directory, simply run: :: 
+     
+    vagrant up 
+ 
+This creates a Virtual Ubuntu 14.04 machine with all the necessary requirements. 
```

Comment:
Perhaps `This creates a VM running Ubuntu 14.04, then automatically installs Girder within it.`?